db/migrate/20220810101622_create_users.rb

```diff
@@ -4,7 +4,7 @@ def change
       t.string :Name
       t.string :Photo
       t.text :Bio
-      t.integer :PostsCounter
+      t.integer :PostsCounter, default: 0
```

Comment:
## [OPTIONAL]

- [ ] Kindly note that the name in convention is extremely important in Ruby on Rails, Only one name in convention error can mess up your code. I would kindly suggest you lowercase and separate words with `_` all of your columns in your tables. For example use `posts_counter` instead of `PostsCounter`, `name` instead of `Name`.......